### PR TITLE
Update ESLint dependencies

### PR DESCRIPTION
**Pull request by abdonrd**
To simplify #1150

### CODE REVIEW

File: constants/appLinks.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ type SegmentData = {
 /**
  * Represent a navigation link for the social media community links of the site
  */
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```
Comment (y4izus) [inline constants/appLinks.ts:8]: We are not using this interface anymore, so please, remove it instead of adding the es-lint comment

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank you!

Just one small change :)

File: constants/appLinks.ts (hunk 1 above)
Comment (abdonrd) [reply constants/appLinks.ts:8]: Perfect! As I am not very familiar with the project, I preferred not to delete anything.

Comment (y4izus) [reply constants/appLinks.ts]: No worries! For me it is the same and I'm familiar with it, but...

Screen Shot 2020-11-26 at 16 59 37

**Review by y4izus — APPROVED** [review]

**Pull request merged**